src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -358,7 +358,7 @@ def find_subscriptions_in_vm_with_msi(self, identity_id=None, allow_no_subscript
         token_entry = msi_creds.token
         token = token_entry['access_token']
         logger.info('MSI: token was retrieved. Now trying to initialize local accounts...')
-        decode = jwt.decode(token, verify=False, algorithms=['RS256'])
+        decode = jwt.decode(token, algorithms=['RS256'], options={"verify_signature": False})
```

Comment:
https://pyjwt.readthedocs.io/en/latest/changelog.html?highlight=verify#dropped-deprecated-verify-param-in-jwt-decode

> Dropped deprecated `verify` param in `jwt.decode(...)`
> Use `jwt.decode(encoded, key, options={"verify_signature": False})` instead.